docs/core_docs/docs/use_cases/media.mdx

```diff
@@ -0,0 +1,56 @@
+# Audio/Video Structured Extraction
+
+Google's Gemini API offers support for audio and video input, along with function calling.
+Together, we can pair these API features to extract structured data given audio or video input.
+
+In the following examples, we'll demonstrate how to read and send MP3 and MP4 files to the Gemini API, and receive structured output as a response.
```

Comment:
Love these examples! Perhaps comment that you don't *need* to use structured output with audio and video, but it always helps to understand what the results can be.